src/types/config/Config.ts

```diff
@@ -4,8 +4,8 @@ import ExportRule from './ExportRule';
 
 export default interface Config {
     path: NormalizedPath;
-    tags: string[];
-    exports: ExportRule[];
-    dependencies: DependencyRule[];
-    imports: string[];
+    tags: string[] | null;
```

Comment:
I think these are actually `undefined`.  (I should probably turn on strict mode for the project...)